tox.ini

```diff
@@ -1,8 +1,8 @@
 [tox]
 envlist=
-    py{38,39,310,311}-core
-    py{38,39,310,311}-lint
-    py{38,39,310,311}-wheel
+    py{38,39,310,311,312}-core
+    py{38,39,310,311,312}-lint
+    py{38,39,310,311,312}-wheel
     py311-wheel-windows
```

Comment:
We should prob change the windows wheel to 3.12 to make sure that's compatible as well. Thoughts there?

Reply:
Guinea Pig PR up: https://github.com/ethereum/eth-rlp/pull/20